src/features/chart/components/Chart.tsx

```diff
@@ -17,33 +17,26 @@ import { ErrorMessage } from "../../error-message";
 import { AppState } from "store";
 import "./Chart.css";
 
-type Tenor = { value: string; label: string };
+export type ChartRange = "1D" | "5D" | "1M" | "1Y" | "5Y" | "MAX";
 
-const tenors: Tenor[] = [
-  { value: "1d", label: "1D" },
-  { value: "5d", label: "5D" },
-  { value: "1m", label: "1M" },
-  { value: "1y", label: "1Y" },
-  { value: "5y", label: "5Y" },
-  { value: "max", label: "MAX" }
-];
+const tenors: ChartRange[] = ["1D", "5D", "1M", "1Y", "5Y", "MAX"];
```

Comment:
nit: I think there could be a better name for this instead of `tenors`, maybe `ChartRange`?